portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +27,40 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+  private final Date startTime = new Date();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Calculate server stats
+    Date currentTime = new Date();
+
+    // Convert data to JSON
+    MarcoStats marcoStats = new MarcoStats(20, "Marco");
+    String json = convertToJsonUsingGson(marcoStats);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+    //response.setContentType("text/html;");
+    //response.getWriter().println("<h1>Hello world!</h1>");
+    //response.getWriter().println("The server's current date is " + new Date());
+  }
+
+   private String convertToJson(MarcoStats marcoStats) {
```

Comment:
Are you using this function anywhere? If not, please delete it (it looks like in the other file you're using 'convertToJsonUsingGson(...)' instead. (but please correct me if I'm wrong)